### PR TITLE
[FT] Added configs for llama 70b on A10

### PR DESCRIPTION
**Pull request by shawnanyscale**
- Added configs for llama 70b on A10
- Will merge after the next deployment

### CODE REVIEW

File: templates/fine-tune-llm/training_configs/full_param/llama-2-70b-4096-4xg5_48xlarge.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+model_id: meta-llama/Llama-2-70b-chat-hf # <-- change this to the model you want to fine-tune
+train_path: s3://air-example-data/gsm8k/train.jsonl # <-- change this to the path to your training data
+valid_path: s3://air-example-data/gsm8k/test.jsonl # <-- change this to the path to your validation data. This is optional
+context_length: 4096 # <-- change this to the context length you want to use
```
Comment (kouroshHakha) [inline templates/fine-tune-llm/training_configs/full_param/llama-2-70b-4096-4xg5_48xlarge.yaml:4]: Have we tested that full param with 4096 context length works on 32 A10s?

File: templates/fine-tune-llm/training_configs/full_param/llama-2-70b-4096-4xg5_48xlarge.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+num_devices: 32 # <-- change this to total number of GPUs that you want to use
+num_epochs: 10 # <-- change this to the number of epochs that you want to train for
+train_batch_size_per_device: 8
```
Comment (kouroshHakha) [inline templates/fine-tune-llm/training_configs/full_param/llama-2-70b-4096-4xg5_48xlarge.yaml:7]: I think bsize is too large for this?

File: templates/fine-tune-llm/training_configs/full_param/llama-2-70b-4096-4xg5_48xlarge.yaml (hunk 1 above)
Comment (shawnanyscale) [reply templates/fine-tune-llm/training_configs/full_param/llama-2-70b-4096-4xg5_48xlarge.yaml:4]: Yes.

32 A10s and this job took 25 hours: [here](https://console.anyscale-staging.com/o/anyscale-internal/jobs/prodjob_ypx284ttd968uj6xw5ncsbsssv)
I have 2 full-param FT jobs running that are on 4 g5.48xlarge that have been running for 20+ hours:
job 1 [here](https://console.anyscale-staging.com/o/anyscale-internal/jobs/prodjob_j118eshfmjrwkwi52kyxt264xw) and job 2 [here](https://console.anyscale-staging.com/o/anyscale-internal/jobs/prodjob_sjtw8s551t3zb7syq2viiwqmhk)

File: templates/fine-tune-llm/training_configs/full_param/llama-2-70b-4096-4xg5_48xlarge.yaml (hunk 2 above)
Comment (shawnanyscale) [reply templates/fine-tune-llm/training_configs/full_param/llama-2-70b-4096-4xg5_48xlarge.yaml:7]: it has worked for the jobs that i submitted so I am keeping it 8 for now

**Review by kouroshHakha — APPROVED** [review]

**Pull request merged**